build-system/npm-publish/write-package-files.js

```diff
@@ -15,16 +14,27 @@ const {log} = require('../common/logging');
 const {stat, writeFile} = require('fs/promises');
 const {valid} = require('semver');
 
+const [extension, ampVersion, extensionVersion] = process.argv.slice(2);
 const packageName = getNameWithoutComponentPrefix(extension);
-const dir = `extensions/${extension}/${extensionVersion}`;
```

Comment:
Why not evaluate `dir` here so you're not calling `getDir()` multiple times.

Reply:
That does make sense.